runtime/testy/runtime.py

```diff
@@ -269,14 +264,14 @@ def runtimeTest(testNames):
   testNameRegex = re.compile(".*_setup")
   allTestNames = [testName[:-len("_setup")] for testName in dir(studentCode) if testNameRegex.match(testName)]
 
-  if len(testNames) == 0:
-    print("Running all tests")
-    testNames = allTestNames
-  else:
+  if testNames:
     for testName in testNames:
       if testName not in allTestNames:
         print("Error: {} not found.".format(testName))
         return
+  else:
```

Comment:
Style thing i picked up over the summer: 

With if/ else chains, you want to structure the code such that the simpler case comes first, and the more complicated case comes second. It's a readibility thing: that way, people can read the simple case and move on to the complicated case while the conditional clause is still fresh in their mind. 

tl;dr change to `if not testNames` and swap the if/else blocks